sklearn/tests/test_common.py

```diff
@@ -57,13 +56,18 @@
     check_transformer_data_not_an_array,
     check_transformer_n_iter,
     check_non_transformer_estimators_n_iter,
-    CROSS_DECOMPOSITION)
+    CROSS_DECOMPOSITION,
+    check_estimators_pfit_general,
```

Comment:
was the renaming of ``partial_fit`` to ``pfit`` for the sake of line lengths? If so, maybe rather remove the "estimators" from the function name?